mne/io/base.py

```diff
@@ -1066,8 +1066,12 @@ def apply_function(self, fun, picks=None, dtype=None,
         dtype : numpy.dtype (default: None)
             Data type to use for raw data after applying the function. If None
             the data type is not modified.
+        channel_wise: bool (default: True)
+            Whether to apply the function to each channel individually. If
+            False, the function will be applied to all channels at once.
```

Comment:
It's probably worth modifying the docstring for `fun` to say that it needs to operate on an array of shape `(n_times,)` if `channel_wise=True` and `(len(picks), n_times)` if `channel_wise=False`